test/support/generator/get-facility-activation-transaction-generator.ts

```diff
@@ -0,0 +1,132 @@
+        BundleStatusShortDescription: firstFacilityActivationTransaction.bundleStatusDesc,
+      },
+      InitiatingUserName: firstFacilityActivationTransaction.initiatingUserName,
+      PostingDate: postingDateTime,
+      BundleMessageList: [
+        {
+          $type: 'FacilityCodeValueTransaction',
+          AccountOwnerIdentifier: firstFacilityActivationTransaction.accountOwnerIdentifier,
+          EffectiveDate: effectiveDateTime,
+          FacilityIdentifier: facilityIdentifier,
+          FacilityTransactionCodeValue: {
+            FacilityTransactionCodeValueCode: firstFacilityActivationTransaction.facilityTransactionCodeValueCode,
+          },
+          FacilityTransactionType: {
+            TypeCode: firstFacilityActivationTransaction.facilityTransactionTypeCode.toString(),
+          },
+          IsDraftIndicator: firstFacilityActivationTransaction.isDraftIndicator,
+          LenderType: {
+            LenderTypeCode: firstFacilityActivationTransaction.lenderTypeCode,
+          },
+          LimitKeyValue: firstFacilityActivationTransaction.limitKeyValue,
+          LimitType: {
+            LimitTypeCode: firstFacilityActivationTransaction.limitTypeCode,
+          },
+          SectionIdentifier: firstFacilityActivationTransaction.sectionIdentifier,
+        },
+      ],
+    };
+
+    const apiFacilityActivationTransaction: GetFacilityActivationTransactionResponseDto = {
+      portfolioIdentifier,
+      facilityIdentifier,
+      bundleStatusCode: firstFacilityActivationTransaction.bundleStatusCode,
+      bundleStatusDesc: firstFacilityActivationTransaction.bundleStatusDesc,
+      lenderTypeCode: firstFacilityActivationTransaction.lenderTypeCode,
+      initialBundleStatusCode: firstFacilityActivationTransaction.initialBundleStatusCode,
+      initiatingUserName: firstFacilityActivationTransaction.initiatingUserName,
+      accountOwnerIdentifier: firstFacilityActivationTransaction.accountOwnerIdentifier,
+      effectiveDate: firstFacilityActivationTransaction.effectiveDate,
+      facilityTransactionCodeValueCode: firstFacilityActivationTransaction.facilityTransactionCodeValueCode,
+      facilityTransactionTypeCode: firstFacilityActivationTransaction.facilityTransactionTypeCode,
+      isDraftIndicator: firstFacilityActivationTransaction.isDraftIndicator,
+      limitKeyValue: firstFacilityActivationTransaction.limitKeyValue,
+      limitTypeCode: firstFacilityActivationTransaction.limitTypeCode,
+      sectionIdentifier: firstFacilityActivationTransaction.sectionIdentifier,
+    };
+
+    return {
+      acbsFacilityActivationTransaction,
+      apiFacilityActivationTransaction,
+    };
+  }
+}
+
+interface FacilityActivationTransactionValues {
+  initialBundleStatusCode: BundleStatusEnum;
+  bundleStatusCode: string;
+  bundleStatusDesc: string;
+  initiatingUserName: string;
+  accountOwnerIdentifier: AcbsPartyId; // TODO: is this right to use acbsPartyId?
```

Comment:
Can we associate a ticket number with this TODO please.

Reply:
Done